src/main/scala/cromwell/binding/values/WdlValue.scala

```diff
@@ -44,4 +46,9 @@ trait WdlValue {
   def collectAsSeq[T <: WdlValue](filterFn: PartialFunction[WdlValue, T]): Seq[T] = {
     if (filterFn.isDefinedAt(this)) Seq(filterFn(this)) else Nil
   }
+
+  /**
+   * @return MD5 Hash.
+   */
+  def getHash(implicit hasher: FileHasher): Hash = valueString.md5Sum
```

Comment:
I don't think it's obvious it returns the *MD5* hash, although maybe we shouldn't be committing to that implementation from an API perspective?  Also we're sort of "rolling our own" hashing on the richer types, although all of that is based on MD5.